### ISSUE TITLE
mounting multiple locations always connects to the first

### ISSUE THREAD

**Issue by lycestra** [macos-fuse-t/sshfs#1]
Mounting the first FS works fine, but any FS mounted after that succeeds yet acts as a mount to the first. Same or different host. Even after unmounting the first, as long as another FS is mounted, the first is retained and used for each filesystem mounted. If all are unmounted, then it will allow a new FS to mount, but continues as the new First for any additional mounted fs. Not sure if this is an sshfs-related handle reuse, or libfuse-t

**Comment by macos-fuse-t** [comment]
Two mounts at the same time?

**Comment by lycestra** [comment]
Correct. Two at the same time. While it appears that multiple processes can coexist, only the first is expressed, and at all the mount points.
e.g:
`sshfs location1 ~/mnt/path1`
`sshfs location2 ~/mnt/path2`

This results in location1 being mounted at both `~/mnt/path1` and `~/mnt/path2`. After unmounting `~/mnt/path1`, location1 is still mounted at `~/mnt/path2`

**Comment by macos-fuse-t** [comment]
Thanks for the report. I will test it.
